Docker/test/selenium/Dockerfile

```diff
@@ -7,4 +7,4 @@ RUN pip3 install --no-cache --upgrade pip setuptools
 
 WORKDIR /usr/src/app/
 COPY . .
-RUN npm ci
\ No newline at end of file
+RUN if [ ! -d "/usr/src/app/node_modules" ]; then npm ci --verbose ; fi
```

Comment:
Do we not need to make sure this is up to date still?

Reply:
I've been commenting this out manually because I have the `node_modules` folder which gets copied and then replaced. In CI this would not occur as `node_modules` is not a part of the repo or the cache. This is only here because I don't want to comment/uncomment this all the time.

So it should only happen locally, and I don't have any other ideas than making sure we would be aware this happening or maybe post a big disclaimer that it's not gonna update dependencies on a run?

Reply:
Hm, maybe add it to `.dockerignore`? I'm really afraid this is going to add potential for very subtle problems.

Reply:
Ok the commit hook felt a bit too intrusive as I would have to keep installing them on manually. How about this env variable for use in `local.env` to make this decision a bit more conscious?

Reply:
install dependencies locally

in local.env

```
SKIP_INSTALL_TEST_DEPENDENCIES=1
```